src/idl_gen_text.cpp

```diff
@@ -69,10 +73,46 @@ bool Print(T val, Type type, int /*indent*/, Type * /*union_type*/,
   return true;
 }
 
+// Print a vector as base64-encoded string
+template<typename T>
+bool PrintVectorBase64(const Vector<T> &v, Type type, int indent,
+                       const IDLOptions &opts, std::string *_text,
+                       const FieldDef *fd) {
+  (void)type;
+  (void)indent;
+  auto b64mode = field_base64_mode(fd);
+  // leave if the Filed doesn't have base64 attribute
+  if (!b64mode || (type.base_type != BASE_TYPE_UCHAR)) return false;
+
+  b64mode |= opts.base64_cancel_padding ? kBase64CancelPadding : 0;
+  // Print a sequence of [uint8_t] as base64-encoded string
+  // type T can by any, not only uint8_t
+  const auto *src = reinterpret_cast<const uint8_t *>(v.data());
+  const auto src_size = v.size();
+  const auto req_size = Base64EncodedSize(b64mode, src, src_size);
```

Comment:
please refactor these functions. `Base64EncodedSize` is not needed, since the destination is a vector (or string), so we can simply `push_back`. There's no need for an intermediate vector either. Just pass `_text` to `Base64Encode`.

I'm going to stop my review here since it is clear my comments from last time haven't been addressed. Please confirm you have addressed all my comments (from both reviews) thruout the code before asking me to review again.